packages/amplify-category-function/src/index.ts

```diff
@@ -221,6 +222,12 @@ export async function handleAmplifyEvent(context, args) {
   context.print.info(`Received event args ${args}`);
 }
 
+export async function lambdaLayerPrompt(context: $TSContext, resources: Array<any>): Promise<void> {
```

Comment:
nit: use $TSAny instead of any (since typing all resources is out of scope)

Reply:
It doesn't return anything do you still want @TSAny?